iree/testing/BUILD

```diff
@@ -20,16 +20,6 @@ package(
     licenses = ["notice"],  # Apache 2.0
 )
 
-cc_library(
```

Comment:
we should keep this - it's nice to be able to hide the required initialization in a single place for when it differs across platforms, and this will get duplicated (I have a pending commit now that uses this)

Reply:
Done!